repo/incubating/flink/versions/0/flink-frameworkversion.yaml

```diff
@@ -53,9 +115,21 @@ spec:
           spec:
             containers:
             - name: taskmanager
-              image: flink:1.7
+              image: kudobuilder/flink:1.7.2
               args:
               - taskmanager
+              - "-Dhigh-availability={{HIGH_AVAILABILITY}}"
+              - "-Dhigh-availability.storageDir=file:///ha"
+              - "-Dstate.savepoints.dir=file:///snapshots/"
+              - "-Dstate.checkpoints.dir=file:///snapshots/"
+              - "-Dhigh-availability.zookeeper.quorum={{ZOOKEEPER_URL}}"
+              - "-Dhigh-availability.zookeeper.path.root={{ZOOKEEPER_PATH}}"
+              - "-Dhigh-availability.cluster-id={{NAME}}"
+              - "-Dtaskmanager.rpc.port=6122"
+              - "-Dtaskmanager.data.port=6121"
+              - "-Dquery.server.port=6125"
+              - "-Dblob.server.port=6124"
+              - "-Dlog.file=/logs/jobmanager.log"
```

Comment:
This might need to be an environment variable.  The logs still say its not correct